.vscode/settings.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Why were these files deleted?  Do they interfere with clippy somehow?  If possible I'd like to keep them around.

Reply:
They were deleted because I failed to realize they were part of the original repo. I had added my own .vscode settings and then removed them without realizing I had removed the ones from you.

Reply:
Fixed with commit [0efd2b4](https://github.com/ixmilia/dxf-rs/pull/42/commits/0efd2b4a70c995f639be694f440151ea7b65a267)